packages/cli-platform-android/src/config/getMainActivity.ts

```diff
@@ -21,7 +21,7 @@ interface IntentFilter {
  * Reads the AndroidManifest.xml file and returns the name of the main activity.
  */
 
-export default function getMainActivity(manifestPath: string): string | null {
+export default function getMainActivity(manifestPath: string): string {
```

Comment:
I'd rather keep this function returning `string | null` and remove the `if (!mainActivity) {` check only